launch/offboard_control_launch.yaml

```diff
@@ -0,0 +1,7 @@
+launch:
+
```

Comment:
Is this meant to launch standalone, or also automatically launch the micro_dds_agent, similar to sensors_combined_listener launchfile?

Reply:
Yes, I was just playing around so far to get a feel for python vs yaml launch. Do you have a preference?